src/easypost.js

```diff
@@ -123,10 +126,16 @@ export default class API {
     } = params;
 
     const req = this.agent[method](this.buildPath(path))
-                  .accept('json')
-                  .set('Content-Type', 'application/json')
-                  .set(API.buildHeaders(headers))
-                  .auth(`${this.key}:`);
+                .accept('json')
+                .set('Content-Type', 'application/json')
+                .set(API.buildHeaders(headers))
+                .auth(`${this.key}:`);
+
+    if (path === 'rating/v1/rates') {
```

Comment:
^^ for some reason, the content type is set to urlencoded even so (the above code is when I comment out the content type change), also the base url is https://api.easypost.com/v2 , but the url required for rating is just https://api.easypost.com/rating/v1/rates